app/boards/shields/romac_plus/romac_plus.overlay

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2020 Pete Johanson, Richard Jones
```

Comment:
Header ditto

Reply:
Updated as a part of [4c09204](https://github.com/zmkfirmware/zmk/pull/198/commits/4c092044ce22f0c60c3bf0ef860ecc543328d5bf)